charts/flyte-binary/values.yaml

```diff
@@ -375,6 +378,10 @@ rbac:
   # extraRules Add additional rules to the ClusterRole
   extraRules: []
 
+crds:
```

Comment:
Not sure I follow, is this section still needed? Similar to how flyte-core handles this, we could leave the `configuration.propeller.createCrd` key as both condition for the flyteworkflow CRD Helm template and as the input for the `create-flyteworkflow-crd` key in the configmap.

Reply:
Changed.
The original intent was to be able to get rid of CRDs installation completely, but that might be too confusing.